test/TestUtilities/Test.Utility/Signing/CertificateUtilities.cs

```diff
@@ -35,5 +38,29 @@ internal static string GenerateRandomId()
         {
             return Guid.NewGuid().ToString();
         }
+
+        public static X509Certificate2 GetCertificateWithPrivateKey(X509Certificate bcCertificate, AsymmetricCipherKeyPair keyPair)
+        {
+#if IS_DESKTOP
+            RSA privateKey = DotNetUtilities.ToRSA(keyPair.Private as RsaPrivateCrtKeyParameters);
+
+            var certificate = new X509Certificate2(bcCertificate.GetEncoded());
+
+            certificate.PrivateKey = privateKey;
+#else
+            RSAParameters rsaParameters = DotNetUtilities.ToRSAParameters(keyPair.Private as RsaPrivateCrtKeyParameters);
+
+            var rsaCsp = new RSACryptoServiceProvider();
+
+            rsaCsp.ImportParameters(rsaParameters);
+
+            var privateKey = rsaCsp;
```

Comment:
Delete this new variable.  Either rename `rsaCsp` to `privateKey` or just pass `rsaCsp` to `CopyWithPrivateKey(…)`.

Reply:
Thanks! Fixed.